artifacts/LogonScript.ps1

```diff
@@ -246,6 +365,9 @@ $aideuserConfig = @"
         "ServicePrincipalName": "aksedge-sp",
         "Location": "$Location",
         "CustomLocationOID":"$CustomLocationOid",
+        "EnableWorkloadIdentity": true,
```

Comment:
Does this work with the Microsoft tenant?
Can you create clusters and validate e2e pass and so on?
From a Monday meeting, the support for Microsoft tenant is being worked on, and expected within a week or so

Reply:
Is Microsoft tenant same with Corp?

Reply:
Yes. The other tenant would be AzureStack
You can find the tenant if you do account show or go to entra

Reply:
i tested in Microsoft tenant with subscription bd7b1ca2-036a-4ca9-990e-79837f39707e https://dev.azure.com/msazure/One/_build/results?buildId=105325085&view=results
and testing in BareMetal now